src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function sortList(roster) {
+  let rosterItems = roster.querySelectorAll('li');
+
+  [...rosterItems] = [...rosterItems].sort((a, b) => {
+    const numberA = convertToNumber(a.dataset.salary);
+    const numberB = convertToNumber(b.dataset.salary);
+
+    return numberB - numberA;
+  });
+
+  for (const item of [...rosterItems]) {
```

Comment:
Replace the for loop with `forEach` for better readability: `[...rosterItems].forEach(item => roster.append(item));`